CodeGeneration/Package.swift

```diff
@@ -5,10 +5,10 @@ import PackageDescription
 let package = Package(
   name: "CodeGeneration",
   platforms: [
-      .macOS(.v10_15),
-    ],
+    .macOS(.v10_15)
+  ],
```

Comment:
Right now we also include `Package.swift` files. Not sure if we should do that.

Reply:
Why not. I don’t see any reason why it shouldn’t be formatted.